builtin/common/vector.lua

```diff
@@ -70,6 +70,15 @@ function vector.direction(pos1, pos2)
 	})
 end
 
+function vector.angle(a, b)
+	local dotp = a.x * b.x + a.y * b.y + a.z * b.z
+	local cpx = a.y * b.z - a.z * b.y
+	local cpy = a.z * b.x - a.x * b.z
+	local cpz = a.x * b.y - a.y * b.x
+	local crossp = math.sqrt(cpx ^ 2 + cpy ^ 2 + cpz ^ 2)
```

Comment:
Better to call this `crossplen` or something similar, as it's not the cross product.